src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig, iter_: int
         self._config: EvaluatorServerConfig = config
         self._ensemble: Ensemble = ensemble
 
-        self._loop = new_event_loop()
+        self._loop = get_running_loop()
```

Comment:
Not sure here as I think evaluator would like to have its own loop, but  it might be just fine :shrug: